app/src/main/res/values/strings.xml

```diff
@@ -281,8 +281,8 @@
   <string name="upload_problem_image">Problems found in image</string>
   <string name="internet_downloaded">Please only upload pictures that you have taken by yourself. Don\'t upload pictures that you have downloaded from the Internet.</string>
   <string name="give_permission">Give permission</string>
-  <string name="use_external_storage">Use external storage</string>
-  <string name="use_external_storage_summary">Save pictures taken with the in-app camera on your device</string>
+  <string name="use_external_storage">Save pictures</string>
```

Comment:
While I agree that "Use external storage" wasn't a great title for the setting, I also think that "Save pictures" sounds too generic. How about "Save in-app shots" or something along those lines?

Reply:
Here's a code suggestion to easily apply the suggested change:

```suggestion
  <string name="use_external_storage">Save in-app shots</string>
```